docs/tutorials/TracingCode.md

```diff
@@ -231,7 +231,7 @@ Recall from the User Guide that the `edit` command has the format: `edit INDEX [
      * {@code JsonSerializableAddressBook}.
      */
     public JsonSerializableAddressBook(ReadOnlyAddressBook source) {
-        persons.addAll(
+        guests.addAll(
             source.getPersonList()
                   .stream()
                   .map(JsonAdaptedPerson::new)
```

Comment:
should getPersonList and JsonAdaptedPerson be changed to getGuestList and JsonAdaptedGuest?